xml2rfc/templates/doc.yaml

```diff
@@ -1779,7 +1779,7 @@ xref:
       the text of the external link.
 
     - If the "sectino" attribute is present, the rendering will for most
```

Comment:
Probably this when unchecked by the spell checker. `"sectino"` should be `"section"`.

Reply:
Well, check-spelling didn't like it, but Google Sheets didn't make a suggestion, so it just sat there... https://github.com/jsoref/xml2rfc/blob/ad317d2b817979fece5cf2c1b1b34732963c0171/.github/actions/spelling/expect.txt#L489

Sometimes I scan through looking for extra things to clean up, and sometimes I settle for just the suggestions that Google Sheets makes `blandk` would probably be `blank`, but as noted below, it's in the changelog so would want to be skipped.

I've posted an updated list of items:
https://github.com/jsoref/xml2rfc/blob/816778d23a5c3855334ea10fe01702110886f879/.github/actions/spelling/expect.txt